src/llama.cpp

```diff
@@ -2661,6 +2664,9 @@ struct llama_context {
     struct ggml_tensor * inp_s_mask;    // F32 [1, n_kv]
     struct ggml_tensor * inp_s_seq;     // I32 [n_kv, n_batch]
 
+    // KQ mask per layer, used by sliding window attention (gemma 2)
+    struct ggml_tensor * inp_KQ_mask_SWA;
```

Comment:
```suggestion
    struct ggml_tensor * inp_KQ_mask_swa;
```

Reply:
Changed in https://github.com/ggerganov/llama.cpp/pull/8227/commits/ed5496fb32e9888abeaa7672aaba4d4251671457